pkg/apis/aerogear/v1alpha1/zz_generated.deepcopy.go

```diff
@@ -1,5 +1,21 @@
 // +build !ignore_autogenerated
 
+/*
```

Comment:
/me wonders why this never updated for me. What version of the operator-sdk cli do you have?

Reply:
I've got `operator-sdk version: v0.7.0+git`, and when I run `make code/run` (which runs the generation stuff), this get removes. lol

Reply:
```
[aliok@localhost ~]$ operator-sdk --version
operator-sdk version v0.2.1
```

Let me update mine :)

Operator SDK guys are releasing like a boss. New minor release every month. Mine was only 5 months old.

Reply:
Ok, now I have the same behavior